src/Misc/layoutbin/ru-RU/strings.json

```diff
@@ -538,6 +538,7 @@
     "ServiceInstalled":  "Служба \"{0}\" успешно установлена",
     "ServiceRecoveryOptionSet":  "Для службы \"{0}\" успешно задан вариант восстановления",
     "ServiceStartedSuccessfully":  "Служба \"{0}\" успешно запущена",
+    "StartServiceDescription": "запустить службу после окончания конфигурации? ([Д]а/[Н]ет)",
```

Comment:
We are adding only English strings, strings for other languages will be translated by the localization team.

Reply:
My bad